hugo/content/contribute.md

```diff
@@ -3,83 +3,279 @@ title: Contribute
+
+
+# Development
+
+## Coding
+
+<!-- Todo: 
+- mention important tasks ? (e.g. [gRPC development](https://github.com/mumble-voip/mumble/pull/3947))
+
+
+-->
+
+We are always looking for helping hands willing to contribute to mumble.
+You'll need a good understanding of **C++** and knowledge in **Qt** is definitely helpful.
 
-# Report Bugs and Issues
+Our source code is hosted on Github:
 
-If you found a bug or issue you can report them in our [issue tracker](https://github.com/mumble-voip/mumble/issues). Please check if a ticket already exists for your issue before creating a new one.
+- [Overview of Projects](https://github.com/mumble-voip/)
 
-# Suggest Changes and Features
+- [Main Mumble](https://github.com/mumble-voip/mumble) (includes the Server and Client)
+  
+- [iOS app](https://github.com/mumble-voip/mumble-iphoneos) (unmaintained; help wanted)
 
-The issue tracker is also used to track [feature requests, which are marked with the `feature-request` label](https://github.com/mumble-voip/mumble/issues?q=is%3Aopen+is%3Aissue+label%3Afeature-request+sort%3Areactions-%2B1-desc).
+**Getting Started**
 
-Please vote with a thumbs up on those you would like to see being fulfilled, describe your use cases or otherwise participate in the discussion on existing tickets. And if you want to request a feature not suggested yet, please create a new feature request ticket possibly with some elaboration and use cases.
+The best way to get into mumble development is to link up with us on IRC ([#mumble-dev on freenode](irc://chat.freenode.net/mumble-dev)).
 
-# Test new features and developments
+### Documentation regarding Development
 
-Apart from our stable releases we also provide development snapshots for testing the current, intermediate state of development. You can download them from our [Downloads page]({{< relref "downloads" >}})).
+- [Contributing Guidelines](https://github.com/mumble-voip/mumble#contributing)
 
-If you find any issues with them please check if the issue has already been reported, add details if you have more, or add a new tracker item if it has not been reported yet.
+- (Work-in-Progress) [C++ Guidelines & tools](https://github.com/mumble-voip/mumble/discussions/4261) 
+
+### Open Issues and ongoing Development
+
+Take a look at [Issues](https://github.com/mumble-voip/mumble/issues) and [Pullrequests](https://github.com/mumble-voip/mumble/pulls) on Github, especially with the following Labels:
+
+- [Good first issue](https://github.com/mumble-voip/mumble/labels/good%20first%20issue)
+- [Help wanted](https://github.com/mumble-voip/mumble/labels/help%20wanted)
+- [Feature Request](https://github.com/mumble-voip/mumble/labels/feature-request)
+
+<!-- mention priority labels, e.g. [P2- Critical](https://github.com/mumble-voip/mumble/labels/priority%2FP2%20-%20Important) -->
+
+Under [Milestones](https://github.com/mumble-voip/mumble/milestones) we list the Issues that are planned to be worked on for the next releases.
```

Comment:
```suggestion
```
No, we no longer do that. There still exists one for 1.4.0 but it's not up-to-date (we should probably remove it)

Reply:
Also not used anymore and what is used instead?

Reply:
Also nothing. The argumentation is basically the same as above.
Furthermore we intend to switch to a more regular release cycle after 1.4.0 has been released. Thus we won't make a release depend on certain things being implemented or not but simply release what's there at that point instead.

Reply:
I would still label things that the team thinks are important.
So contributers know what is more useful etc.

Also it could be labeled what issues you agree or disagree with, so that no one tries to implement something that you will not accept in the end.